src/components/ProjectHeadline.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default function ProjectHeadline({ title, country }) {
+  return (
+    <Headline>
```

Comment:
Is it really necessary to wrap the h3 in a section? Do you really need a flex-container if it only contains one element?

Reply:
Thanks, changed it